packages/serve/src/lib/document-server/redocDocumentServer/redocDocumentServer.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  DocumentOptions,
+  DocumentServerType,
+  DocumentSpec,
+  ProjectOptions,
+  TYPES as CORE_TYPES,
+  VulcanExtensionId,
+  VulcanInternalExtension,
+} from '@vulcan-sql/core';
+import * as nunjucks from 'nunjucks';
+import { Next } from 'koa';
+import * as Router from 'koa-router';
+import { DocumentServer } from '@vulcan-sql/serve/models';
+import * as fs from 'fs';
+import * as path from 'path';
+import { inject } from 'inversify';
+
+@VulcanInternalExtension('redoc')
```

Comment:
Follow by above `documentServerMiddleware` suggestion. Here the `@VulcanInternalExtension('redoc')` may could change to `documentServerMiddleware` internal extension module name ?

Reply:
it makes sense to merge the options of redoc with 'api-doc' options because it is a built-in router, but I'll prefer using internal options instead of module configuration.